exporters/zipkin/src/test/java/io/opentelemetry/exporter/zipkin/ZipkinTestSpan.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.zipkin;
+
+import static org.mockito.Mockito.mock;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.api.trace.TraceFlags;
+import io.opentelemetry.api.trace.TraceState;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.testing.trace.TestSpanData;
+import io.opentelemetry.sdk.trace.data.EventData;
+import io.opentelemetry.sdk.trace.data.StatusData;
+import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
+import java.net.InetAddress;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import zipkin2.Endpoint;
+import zipkin2.Span;
+
+class ZipkinTestSpan {
+
+  static final String TRACE_ID = "d239036e7d5cec116b562147388b35bf";
+  static final String SPAN_ID = "9cc1e3049173be09";
+  static final String PARENT_SPAN_ID = "8b03ab423da481c5";
+
+  static final InetAddress localAddressForTesting = mock(InetAddress.class);
+
+  private static final Attributes attributes = Attributes.empty();
+  private static final List<EventData> annotations =
+      Collections.unmodifiableList(
+          Arrays.asList(
+              EventData.create(1505855799_433901068L, "RECEIVED", Attributes.empty()),
+              EventData.create(1505855799_459486280L, "SENT", Attributes.empty())));
+
+  private ZipkinTestSpan() {}
+
+  static TestSpanData.Builder buildStandardSpan() {
```

Comment:
#nit: the naming of methods in this class would benefit from more consistency. How about:
-  `buildStandardSpan()` => `spanBuilder()`
- `standardZipkinSpanBuilder(Span.Kind)` => `zipkinSpanBuilder(Span.Kind, InetAddress)`.
-  `buildZipkinSpan(Span.Kind)` can maybe become `zipkinSpan(Span.Kind)`, but it has pretty low utility so you might consider removing it.

Reply:
these names existed before, but I will change.